tests/scripts/fluentd_e2e.sh

```diff
@@ -30,8 +31,8 @@ function build_image() {
 }
 
 function start_fluent_operator() {
-  cd $PROJECT_ROOT 
-  cat manifests/setup/setup.yaml | sed  "s#kubesphere/fluent-operator:latest#kubesphere/fluent-operator:${IMAGE_TAG}#g"  | kubectl apply -f -
+  cd $PROJECT_ROOT
```

Comment:
file https://github.com/fluent/fluent-operator/blob/master/VERSION should be changed too